docs/how-to-guides/how-to-integrate-autoware-with-your-vehicle.md

```diff
@@ -0,0 +1,154 @@
+
+## 2. Create maps
+
+You need both a pointcloud and a vector map to take full advantage of Autoware.
+Since mapping algorithm such as SLAM (simultaneous localization and mapping) is not implemented in the current Autoware, you may need to use 3rd party tools for this step.
+
+### Create a pointcloud map
+
+Use 3rd party tools such as the LiDAR-based SLAM package to create a pointcloud map. Autoware supports the `.pcd` format for this map.
+
+### Create vector map
+
+Autoware supports lanelet2 format for a vector map. Use 3rd party tools or [Vector Map Builder](https://tools.tier4.jp/) to get the `.osm` file.
+
+## 3. Create your meta-repository
+
+A recommended way to integrate Autoware with your real vehicle is to create a meta-repository for the vehicle.
+Create a forked repository of [autowarefoundation/autoware](https://github.com/autowarefoundation/autoware) (we refer to this as meta-repository) and clone the repository.
+
+```bash
+git clone git@github.com:YOUR_NAME/autoware.YOURS.git
+```
+
+## 4. Create the description packages of your vehicle
+
+Next, you need to create description packages that define the vehicle and sensor configuration of your vehicle.
+Once it is done, you can launch your vehicle model by specifying `vehicle_model:=YOUR_VEHICLE` `sensor_model:=SAMPLE_SENSOR_KIT` in the autoware launchers.
+
+Create the following two packages:
+
+- YOUR_VEHICLE_launch (see [here](https://github.com/autowarefoundation/sample_vehicle_launch) for example)
+- YOUR_SENSOR_KIT_launch (see [here](https://github.com/autowarefoundation/sample_sensor_kit_launch) for example)
+
+It is recommended you write the above two packages in `autoware.repos` file of your meta-repository.
+
+### Adapt YOUR_VEHICLE_launch for autoware launching system
+
+#### At YOUR_VEHICLE_description
+
+Define URDF and parameters in the package (see [here](https://github.com/autowarefoundation/sample_vehicle_launch/tree/main/sample_vehicle_description) for example).
+
+#### At YOUR_VEHICLE_launch
+
+Create a launch file (see [here](https://github.com/autowarefoundation/sample_vehicle_launch/tree/main/sample_vehicle_launch) for example).
+If you have multiple vehicles with similar hardware setup, you can specify `vehicle_id` to distinguish them.
+
+### Adapt YOUR_SENSOR_KIT_description for autoware launching system
+
+#### At YOUR_SENSOR_KIT_description
+
+Define URDF and extrinsic parameters for all the sensors here (see [here](https://github.com/autowarefoundation/sample_sensor_kit_launch/tree/main/sample_sensor_kit_description) for example).
+Note that you need to calibrate extrinsic parameters for all the sensors beforehand.
+
+#### At YOUR_SENSOR_KIT_launch
+
+Create `launch/sensing.launch.xml` that launches the interfaces of all the sensors on the vehicle. (see [here](https://github.com/autowarefoundation/sample_sensor_kit_launch/tree/main/sample_sensor_kit_launch) for example).
+
+!!! note
+
+    At this point, you are now able to run planning_simulator.
```

Comment:
I am assuming that the purpose of running the planning simulator is to test the vehicle and sensing packages.

```suggestion
    At this point, you are now able to run Autoware's Planning Simulator to do a basic test of your vehicle and sensing packages.
```